File: Ejercicio_DOCKER/Ejercicio Docker/Circulo.java
import java.util.Scanner;

//Dado el radio de un cIrculo, calcular su perImetro y su área
public class Circulo {
	public static void main(String[] args) {
		Scanner tec = new Scanner(System.in);
		
		System.out.println("\n\n");
		System.out.println("AREA Y PERIMETRO DEL CIRCULO");
		System.out.println("----------------------------");
		
		double radio, area, perimetro;
		System.out.print("Radio: ");
		radio = tec.nextDouble();
		
		area = Math.PI * radio * radio;
		perimetro = 2 * Math.PI * radio;
		
		System.out.println("Area: " + area);
		System.out.println("Perimetro: " + perimetro);
		System.out.println("\n\n");

	}
}

File: Ejercicios Programación/javascript/Ejercicio_DOCKER/circulo.js
// main.js
const read = require('./leer');

console.log(`\n\n`);
console.log(`AREA Y PERIMETRO DEL CIRCULO`);
console.log(`----------------------------`);
let radio = read("Radio: ");
let area = Math.PI * radio * radio;
let perimetro = 2 * Math.PI * radio;
console.log(`Area: ${area}`);
console.log(`Perimetro: ${perimetro}`);
console.log(`\n\n`);